src/config/colors.js

```diff
@@ -12,6 +12,7 @@ export const colors = {
   greyMidLight: '#f6f8fa',
   greyVeryLight: '#fbfbfb',
   lightGreen: '#00FF00',
+  paleGreen: '#359d68',
```

Comment:
Curious, do you think `paleGreen` will look better than `green` here? Thinking a lighter shade bolded might give it more pop 🎉 but maybe that's just me 😂